irteus/irtgeo.l

```diff
@@ -88,6 +88,19 @@
         (setq dif-rot 
               (transform (transpose (send self :worldrot))
                          (scale (acos (v. a0 a1)) (normalize-vector (v* a0 a1))))))
+       ((:xy :yx :yz :zy :zx :xz)
+        (setq a0 (send self :axis (case rotation-axis ((:xy :yx) :z) ((:yz :zy) :x) ((:zx :xz) :y)))
```

Comment:
How about use 'let'. The following code is my suggestion.
```
((:xy :yx :yz :zy :zx :xz)
        (let ((ax1 (case rotation-axis ((:xy :yx) :z) ((:yz :zy) :x) ((:zx :xz) :y)))
              (ax2 (case rotation-axis ((:xy :yx) :x) ((:yz :zy) :y) ((:zx :xz) :z))))
          (setq a0 (send self :axis ax1)
                a1 (send coords :axis ax1))
          (setq dif-rot
                (transform (transpose (send self :worldrot))
                           (scale (acos (v. a0 a1)) (normalize-vector (v* a0 a1)))))
          (let ((self-coords (send (send self :copy-worldcoords) :rotate (norm dif-rot) dif-rot)))
            (setq a0 (send self-coords :axis ax2)
                  a1 (send coords :axis ax2))
            (setq dif-rot
                  (transform (transpose (send self-coords :worldrot))
                             (scale (acos (v. a0 a1)) (normalize-vector (v* a0 a1))))))))
```

Reply:
Thank you. I fixed. https://github.com/euslisp/jskeus/pull/568/commits/815f8b46e9b7ed047e85a4a8aa8972f93dbc449c